biggraphite/accessor.py

```diff
@@ -562,10 +564,13 @@ class Accessor(object):
 
     __metaclass__ = abc.ABCMeta
 
+    max_requests_per_connection = None
+
     def __init__(self, backend_name):
         """Set internal variables."""
         self.backend_name = backend_name
         self.is_connected = False
+        self._downsampler = _downsampling.Downsampler(accessor=self)
```

Comment:
Same thing, not all drivers will use the software downsampler, some might later have database primitives to do that in the database directly